_posts/2022-09-19-public-jenkins.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "OpenSearch automated build system is now live"
+authors: 
+  - sayaligaikawad
+date: 2022-09-14 01:01:01 -0700
+categories: 
+  - community
+---
+
+OpenSearch project deployed automated build, test, and release infrastructure that supports the end-to-end lifecycle of public distributions along with the launch of 2.3.0 version. This includes the migration of OpenSearch Gradle check workflows that are executed on every PR merge, to the public infrastructure. Community users can now view logs by accessing the [build infrastructure](https://build.ci.opensearch.org/) to understand the current state of their workflows and monitor runs as needed.
+
+This build system will act as a continuous integration and continuous deployment (CI / CD) system that triggers the generation of distribution builds along with automated execution of integration, backward compatibility and performance tests for every build. The enhanced process will also automatically publish the artifacts to target distribution channels including but not limited to Docker, S3, cloud front, Maven etc. The plugin developers can on-board their plugin artifacts in to OpenSearch distributions by adding the component to the distribution input manifest file. The onboarding plugin will need to provide information such as GitHub repository url, the branch which needs to be built, what platforms does the plugin support and checks to run on the component such as version check, dependency check, etc. See [example](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.3.0/opensearch-1.3.0.yml#L29-L34). The end to end build process will use the information on the file to build, assemble, test and publish the artifacts to corresponding distribution targets.
```

Comment:
```suggestion
This build system will act as a continuous integration and continuous deployment (CI / CD) system that triggers the generation of distribution builds along with automated execution of integration, backward compatibility and performance tests for every build. The enhanced process will also automatically publish the artifacts to target distribution channels including but not limited to Docker, S3, cloud front, Maven etc. The plugin developers can on-board their plugin artifacts in to OpenSearch distributions by adding the component to the distribution input manifest file. The onboarding plugin will need to provide information such as GitHub repository url, the branch which needs to be built, what platforms does the plugin support and checks to run on the component such as version check, dependency check, etc. See [example](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.3.0/opensearch-1.3.0.yml#L29-L34). The end to end build process will use the information on the file to build, assemble, test and publish the artifacts to corresponding distribution targets.
```
Change first sentence to "This build system acts as a..."

Change second sentence to "The enhanced process also automatically publishes..."

Change "such as GitHub repository url" to "GitHub repository URL"

Change "what platforms does the plugin support" to "which platforms does the plugin support"

Change "The end to end build process will use" to "The end to end build process uses"